docs/tree_transormations.md

```diff
@@ -22,6 +22,7 @@ Here is the table of supported transformations for each language.
 | Compress operators          | ✅    | ❓      |
 | Hide literals               | ✅    | ⚙️      |
 | Remove comments             | ✅    | ⚙️      |
+| Exclude asterisks            | ✅    | ❓️      |
```

Comment:
I suggest adding this line at bottom of the table, to keep a timeline of adding new transformations.
Also, don't forget to describe this transformation below, from the name it doesn't clear what purpose it.